### ISSUE TITLE
Export (and document) activities that could be easily invoked by Automation, Easer, Tasker and Termux

### ISSUE THREAD

**Issue by xandro0777** [Myzel394/Alibi#93]
Thinking about
* start recording
* export(save) to file(s)
* stop recording
* delete internal recordings

Since all scripted solutions should be able to call several activities consecutively it might be best to keep those activities as simple as possible, eg not combining export and stop.

This should help people deal with a wide array of situations like start recording at boot or when leaving home or entering car and many more.

**Comment by Myzel394** [comment]
I was thinking about this, but just yesterday I got the idea of adding plugins instead (see my comment https://github.com/Myzel394/Alibi/issues/15#issuecomment-2002611208) - this would allow more granular control as the plugin can also react to audio events for example. What do you think about that?

**Comment by xandro0777** [comment]
I have never done that but imagine that exporting the activities so that Easier etc can call them would be many times easier than plugins? If so it looks like a good strategy to do the low hanging fruits first.

**Comment by Myzel394** [comment]
> * export(save) to file(s)
* stop recording
* delete internal recordings

Unfortunately,. it's pretty hard / impossible to record in background due to Android's security. As far as I know you cannot start recording from the background due to privacy reasons. Maybe there's some exception for those automation apps though, I'll take a look.

However, the other activities are fairly easy to implement, I'll work on that! :)

**Comment by xandro0777** [comment]
I think those automation apps can invoke any activity, be it foreground or background?

**Comment by atomicwrites** [comment]
Yeah you can start a foreground activity with most automation apps and that would be fine IMO, I would like to auto trigger when plugged in and connected to my car's Bluetooth and it's fine if the app pops in to the foreground.
